### PR TITLE
navbar-footer

### PR DESCRIPTION
**Pull request by EleMashige**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by TresorBipa — APPROVED** [review]
Good job Ele, Is like the login button  is not yet done? if yes that's its okay. Good job for the start point

**Comment by FatemRahimi** [comment]
Hi ele thank you for your pull request. It looks like nice

File: client/src/App.js
Comment (Mathias02) [inline client/src/App.js:1]: Well done @EleMashige  even though I am unable to see it in the browser but it looks good

Comment (EleMashige) [reply client/src/App.js:1]: Can you please approve?

**Review by TresorBipa — APPROVED** [review]

**Review by Mathias02 — APPROVED** [review]

**Pull request merged**